packages/headless/src/controllers/quickview/headless-quickview.test.ts

```diff
@@ -131,4 +133,12 @@ describe('Quickview', () => {
 
     expect(quickview.state.isLoading).toBe(false);
   });
+
+  it(`when the resultPreview is initialized,
+  #options.maximumPreviewSize is 0`, () => {
```

Comment:
I feel like `undefined` or `null` would be more explicit than `0`, although others may disagree.

Reply:
We initialize it to 0 so that if it is not defined, the API will still return the full sized documents in the quickview.

Reply:
0 is the correct value indeed.